.ci_support/linux_64_.yaml

```diff
@@ -1,7 +1,7 @@
 aws_crt_cpp:
 - 0.29.9
 aws_sdk_cpp:
-- 1.11.488
+- 1.11.458
```

Comment:
Why is it downgrading the aws-sdk-cpp pin? Need to investigate

Reply:
Ok, this is expected behavior. The aws-sdk-cpp 1.11.488 migration was rolled back because it broke arrow

xref: https://github.com/conda-forge/aws-sdk-cpp-feedstock/issues/924, https://github.com/conda-forge/arrow-cpp-feedstock/pull/1682#issuecomment-2601188588, https://github.com/apache/arrow/issues/45304, https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/6938, https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/6942